django_project/frontend/src/components/Navbar/index.jsx

```diff
@@ -75,9 +75,17 @@ export function GeoRepoIndicator() {
 
 export default function NavBar({ minified }) {
   const helpPageRef = useRef(null);
-  const { icon, favicon, site_title } = preferences;
+  const { icon, favicon, site_title, site_type } = preferences;
   const { is_contributor } = user;
   const user_permission = useSelector(state => state.dashboard?.data?.user_permission);
+  const pageTitle = document.title;
+  let headerLink = site_title;
+  if (site_type === 'Staging') {
```

Comment:
You can use css instead this hardcoded
Please check here : https://github.com/unicef-drp/GeoSight-OS/blob/main/django_project/frontend/src/pages/Admin/Components/SideNavigation/style.scss#L66

Just add a span with a class and add that class name on above css